docs/wtf-is-metagame/origin-story.md

```diff
@@ -0,0 +1,173 @@
+---
+title: 📖 The MetaGenesis Odyssey
+---
+
+## MetaGenesis Odyssey  
```

Comment:
I think you can remove this bit since it already shows up in the title above. Also, wondering if we should mention this page is about the "MetaGame Origin", some people might not understand what "The MetaGenesis Odyssey" means or what the purpose of this page in the docs is.